@remirror/react/src/components/__tests__/remirror-manager.spec.tsx

```diff
@@ -93,10 +93,7 @@ describe('manager prop', () => {
 test('it supports <RemirrorExtension />', () => {
   const Component: FC = () => {
     const manager = useRemirrorManager();
-    expect(manager.extensions).toContainAnyValues([
-      expect.any(NewExtension),
-      expect.any(PlaceholderExtension),
-    ]);
+    expect(manager.extensions).toContainValues([expect.any(NewExtension), expect.any(PlaceholderExtension)]);
```

Comment:
`toContainAnyValues` would pass if _either_ extension was available. Fixed this test to `toContainValues`, which requires all of them.